addons/main/settings.sqf

```diff
@@ -221,6 +221,16 @@ _curCat = LSTRING(Settings_Debug);
 
         GVAR(debug_drawRectCacheCurator) = [];
         GVAR(debug_drawRectInUseCurator) = [];
+        if (GVAR(debug_Drawing)) then {
```

Comment:
Wouldn't it be cleaner to use the execution blocks `params ["_value"];` here?

Reply:
I forgot how the parameters of the settings callback worked